ospf6d/ospf6_main.c

```diff
@@ -173,6 +173,37 @@ FRR_DAEMON_INFO(ospf6d, OSPF6, .vty_port = OSPF6_VTY_PORT,
 		.n_yang_modules = array_size(ospf6d_yang_modules),
 );
 
+/* Max wait time for config to load before accepting hellos */
+#define OSPF6_PRE_CONFIG_MAX_WAIT_SECONDS 600
+
+static void ospf6_config_finish(struct event *t)
+{
+	zlog_err("OSPF6 configuration end timer expired after %d seconds.",
```

Comment:
Not sure, why it's an error? Sounds more like a notice or a warning.

Reply:
If we haven't received a configuration end in 10 minutes, it would seem this is an error. We can discuss further in the meeting on Tuesday.